src/class-scripts.php

```diff
@@ -41,9 +41,12 @@ public function __construct( Parsely $parsely ) {
 	 * @return void
 	 */
 	public function run(): void {
-		add_action( 'init', array( $this, 'register_scripts' ) );
-		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_js_api' ) );
-		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_js_tracker' ) );
+		$parsely_options = $this->parsely->get_options();
+		if ( ! $this->parsely->api_key_is_missing() && ! $parsely_options['disable_javascript'] ) {
```

Comment:
Consider using `false ===` instead of `!`, and simplifying `! $this->parsely->api_key_is_missing()` by using the `api_key_is_set()` function.

Reply:
Nice catch